apps/src/lib/kits/maker/toolkit.js

```diff
@@ -32,7 +33,7 @@ import {getAppOptions} from '@cdo/apps/code-studio/initApp/loadApp';
 export {dropletConfig, configMicrobit, configCircuitPlayground, MakerError};
 
 /**
- * @type {CircuitPlaygroundBoard} The current board controller, populated when
+ * @type {CircuitPlaygroundBoard | FakeCPBoard || MicroBitBoard || FakeMBBoard} The current board controller, populated when
```

Comment:
Super tiny NIT: should there be another vertical line between CircuitPlaygroundBoard | FakeCPBoard?

Reply:
Thanks! I mean to be a single vertical line in-between the classes as it is [here](https://github.com/code-dot-org/code-dot-org/blob/9b40ce2be6aa137d72c14973f8acb717bae5c4d4/apps/src/lib/kits/maker/commands.js#L14).

Reply:
👍